apps/wing-console/console/app/demo/main.w

```diff
@@ -106,6 +107,7 @@ let table = new ex.Table(
 let rateSchedule = new cloud.Schedule(cloud.ScheduleProps{
   rate: 5m
 }) as "Rate Schedule";
+nodeof(rateSchedule).collapsible = false;
```

Comment:
```suggestion
nodeof(rateSchedule).expanded = true;
```

I would use `expanded` instead of `collapsable`

Reply:
done 👍